opencti-platform/opencti-front/src/private/components/common/form/CreatorField.tsx

```diff
@@ -73,7 +73,11 @@ const CreatorField: FunctionComponent<CreatorFieldProps> = ({
           label: n?.node.name,
           value: n?.node.id,
         }));
-        const templateValues = [...creators, ...NewCreators];
+        const emptyValue = {
+          label: '',
+          value: '',
+        };
+        const templateValues = [emptyValue, ...creators, ...NewCreators];
```

Comment:
Exact same comments as @SouadHadjiat. I also have that empty field in the autocomplete drawer and if I put an empty field, when I reopen the update, my old user is still here.
Maybe add a "null" in your onInputChange={searchCreators || null} ?

Reply:
@SarahBocognano @SouadHadjiat, after more investigations on that, the issue comes from validation schema for the creator only.

It's ready for another review :)